sdk/streamlib-python-wheel/tests/test_encoded_video_frame_cast.py

```
@@ -249,6 +249,17 @@ def test_a_malformed_color_names_this_bag_rather_than_a_video_frame():
         EncodedVideoFrame.from_bag(encoded_frame_bag(color="bt709"))
 
 
+def test_an_unplaceable_colour_name_is_refused_naming_this_bag_and_the_axis():
+    """The same reader serves both casts, so the refusal must carry this
+    bag's own key — `color`, never `color_info` — beside the axis and value."""
+    with pytest.raises(
+        ValueError,
+        match="bag is not an encoded video frame: color.transfer is 'bt_709', "
+        "which is not an H.273 transfer name",
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)